### DISCUSSION TITLE
I'm facing this issue `swapon: command not found`

### DISCUSSION THREAD

**Discussion by ezzaldeeen** [apache/doris#33638] [RFC]
Hi, I'm using MacOS m1

I have installed the compiled version **(2.0.8 stable) arm64**. and i got this issue when trying to execute:
```bash
./bin/start_be.sh
./start_be.sh: line 81: swapon: command not found  # issue
```

**swapon command** isn't supported by MacOS

the `start_fe.sh` works fine.

**Comment by yagagagaga** [comment]
Maybe you can run Doris on Docker and modify start_be.sh and remove exit 1 after the swap check.
```bash
if [[ "$(swapon -s | wc -l)" -gt 1 ]]; then
    echo "Please disable swap memory before installation."
    exit 1 // remove it. it will impact  performance. but normal verification still can be done.
fi
```

**Reply by ezzaldeeen** [reply]
thank you

**Reply by issue-K** [reply]
Does that mean it can't run on m1 and must be compiled in docker? Does it matter if I remove this code after compiling it in m1 (it doesn't seem to work after trying it O O)
